calendar/calendar-summary-tool/tool/src/java/org/sakaiproject/tool/summarycalendar/ui/CalendarBean.java

```diff
@@ -304,7 +310,7 @@ private CalendarEventVector getEventsFromSchedule() {
 				lastDay.set(Calendar.MILLISECOND, 999);
 				dayOfWeek = lastDay.get(Calendar.DAY_OF_WEEK);
 				// TODO Allow dynamic choice of first day of week
-				while(dayOfWeek != Calendar.SATURDAY){
+				while(dayOfWeek != Calendar.SUNDAY){
```

Comment:
I'm slightly concerned about this for non Monday languages, but we'll see in testing. Otherwise it looks good and core team agrees.